Makefile

```diff
@@ -35,19 +37,29 @@ help:
 	@echo
 	@echo 'Configuration options:'
 	@echo
-	@echo '  GITHUB_USER       Which GitHub user account to use for cloning? [default=anonymous]'
-	@echo '  MINIKUBE_DRIVER   Which Minikube driver to use? [default=kvm2]'
-	@echo '  MINIKUBE_PROFILE  Which Minikube profile to use? [default=minikube]'
-	@echo '  TIMECHECK         Checking frequency in seconds when bringing cluster up and down? [default=5]'
-	@echo '  TIMEOUT           Maximum timeout to wait when bringing cluster up and down? [default=300]'
-	@echo '  VENV_NAME         Which Python virtual environment name to use? [default=reana]'
-	@echo '  DEMO              Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  GITHUB_USER         Which GitHub user account to use for cloning? [default=anonymous]'
+	@echo '  MINIKUBE_DRIVER     Which Minikube driver to use? [default=kvm2]'
+	@echo '  MINIKUBE_PROFILE    Which Minikube profile to use? [default=minikube]'
+	@echo '  TIMECHECK           Checking frequency in seconds when bringing cluster up and down? [default=5]'
+	@echo '  TIMEOUT             Maximum timeout to wait when bringing cluster up and down? [default=300]'
+	@echo '  VENV_NAME           Which Python virtual environment name to use? [default=reana]'
+	@echo '  DEMO                Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  REANA_CLUSTER_MODE  REANA Cluster environment mode [stable|latest|dev, default=latest]'
```

Comment:
Maybe call it `CLUSTER_CONFIG`?  We don't need leading REANA since other variables don't have them, and better to say "configuration" rather than "mode".  In `reana-cluster` help we call `-f` option "REANA cluster specifications file describing configuration for the cluster and for REANA components".  Perhaps we can use the same terminology "cluster configuration file" everywhere...

In We use `-f`